x-pack/plugins/transform/common/constants.ts

```diff
@@ -190,6 +190,17 @@ export const TRANSFORM_HEALTH_CHECK_NAMES: Record<
       }
     ),
   },
+  healthCheck: {
+    name: i18n.translate('xpack.transform.alertTypes.transformHealth.healthCheckName', {
+      defaultMessage: 'Unhealthy transforms',
```

Comment:
To be consistent with the name for the other check, this should probably be `Unhealthy transform`. Any thoughts @szabosteve ?

<img width="481" alt="image" src="https://user-images.githubusercontent.com/7405507/223091698-1fcf6ea0-0926-492e-a47c-0acdd74798df.png">

Reply:
Yes, for the sake of consistency, the noun in the second title should be singular, too: `Unhealthy transform`.

Reply:
Fixed in https://github.com/elastic/kibana/pull/152561/commits/e5afe623dad12c254ad3e85e1dba6dcd2febb737